hunting-buddy.lic

```diff
@@ -107,7 +107,9 @@ class HuntingBuddy
 
   def find_hunting_room?(zone_name)
     UserVars.friends = @settings.hunting_buddies || []
-    if rooms = @hunting_zones[zone_name]
+    rooms = []
+    rooms = Array(zone_name).each { |name| rooms.push(*@hunting_zones[name]).flatten; } 
```

Comment:
I think it'll work as
`rooms = @hunting_zones.values_at(*Array(zone_name)).flatten`